src/components/Form/FormSection.tsx

```diff
@@ -93,15 +93,15 @@ export function FormSection<TData extends GenericFormValues>({
       event && event.preventDefault();
 
       if (onSubmit) {
-        let response: { errors?: string[] };
+        let response: { errors?: string[] } | void;
 
         try {
           setFormState((s) => {
             return { ...s, isBusy: true };
           });
           response = await onSubmit(values);
         } catch (error: unknown) {
-          response = { errors: Array.of(error instanceof Error ? error.message : (error as string)) };
+          response = { errors: Array.of(error as string) };
```

Comment:
OK, I think all the changes in this file should be rolled back. Looks like it's all old stuff from old branch.

Reply:
yes, rolled back